src/resolvers/query/platform-releases/index.js

```diff
@@ -0,0 +1,40 @@
+import fragment from "./fragment";
+import log from "logger";
+import { addFragmentToInfo } from "graphql-binding";
+import config from "config";
+import { size, first } from "lodash";
+import semver from "semver";
+/*
+ * Get list of available platform versions.
+ * @param {Object} parent The result of the parent resolver.
+ * @param {Object} args The graphql arguments.
+ * @param {Object} ctx The graphql context.
+ * @return {PlatformRelease} PlatformRelease.
+ */
+export default async function updateAvailable(parent, args, ctx, info) {
+  // Get current helm release version
+  const currentVersion = config.get("helm.releaseVersion");
+
+  // Get all available versions from database
+  const platformReleases = await ctx.db.query.platformReleases(
+    {},
+    addFragmentToInfo(info, fragment)
```

Comment:
```suggestion
    info
```

No fragment needed/wanted here.

Reply:
Actually, correction: we maybe do, but the only field we need to explicitly ensure is `version`.